### ISSUE TITLE
accept a path or `bytes` for `Single` and `Parallel`

### ISSUE THREAD

**Issue by rfdzan** [rfdzan/jippigy#21]
```rs
let _ = Single::from_bytes(); // accepts a vector (or array) of bytes 
let _ = Single::from_path(); // accepts a path leading to a .jpg image

let _ = Parallel::from_dir(); // accepts a directory containing .jpg images
let _ = Parallel::from_vec(); // accepts a vector of `Vec<u8>`'s or `&[u8]`'s
```

This allows for broader use of the crate.

**Comment by rfdzan** [comment]
it is decided this crate will not interact with filesystem at all, it will only deal with image bytes.

**Comment by rfdzan** [comment]
closed in #23
